src/constants.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""This module defines constants used throughout the Synapse application."""
+
+CHECK_ALIVE_NAME = "synapse-alive"
+CHECK_MJOLNIR_READY_NAME = "synapse-mjolnir-ready"
+CHECK_NGINX_READY_NAME = "synapse-nginx-ready"
+CHECK_READY_NAME = "synapse-ready"
+COMMAND_MIGRATE_CONFIG = "migrate_config"
+SYNAPSE_CONFIG_DIR = "/data"
+MJOLNIR_CONFIG_PATH = f"{SYNAPSE_CONFIG_DIR}/config/production.yaml"
+MJOLNIR_HEALTH_PORT = 7777
```

Comment:
`MJOLNIR_HEALTH_PORT` and perhaps other constants are used only in tests. They shouldn't be here

Reply:
Related: #52